src/promptix/core/components/prompt_loader.py

```
@@ -0,0 +1,129 @@
+            StorageError: If prompt is not found.
+        """
+        prompts = self.get_prompts()
+        if prompt_template not in prompts:
+            available_prompts = list(prompts.keys())
+            raise StorageError(
+                f"Prompt template '{prompt_template}' not found",
+                details={
+                    "prompt_template": prompt_template,
+                    "available_prompts": available_prompts
+                }
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)